packages/orama/src/components/index.ts

```diff
@@ -67,9 +75,28 @@ export type VectorIndex = {
   }
 }
 
+export const enum TreeType {
```

Comment:
Don't use enums in TypeScript. You don't have control over their compilation in JavaScript.